contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+
+If a node's balance with a peer crosses the payment threshold, which is a number every node can set individually (see `swap-payment-threshold` flag), but has a reasonable default defined in the code, then the node kicks off the settlement process. This process involves a series of security and sanity checks, culminating in sending a **signed cheque** to its peer. This signed cheque is a piece of data containing the amount, the source chequebook address and the beneficiary chequebook address, as well as the signature of the cheque issuer. 
+
+The peer, upon receiving the cheque, resets the balance with the cheque value and will initiate a cashing transaction trying to cash the cheque in (considering it makes economic sense as described below) - this is a transaction on a blockchain and represents real financial value. If the cheque was valid and backed by funds, this results in a real transfer of funds from the issuer's contract address to the beneficiary's. All of the accounting is maintained per peer and thus threshold crossing and cheque issuance is based on every individual balance with every peer.
+
+.. image:: img/cheque.svg
+   :alt: flow diagram depicting how cheques are triggered 
+   :width: 400
+
+Interaction with the blockchain
+-------------------------------
+A cheque is sent as part of its own lower layer transport protocol (currently on top of `devp2p`, with the planned transition of the whole of Swarm to `libp2p`). The receiving peer handles the cheque and tries to cash it by issuing a transaction on the blockchain. 
+
+Every node has to deploy its own instance of the Swarm smart contract, also often referred to as *chequebook*.  So when receiving a cheque, the beneficiary issues a transaction *on the contract of the issuer of the cheque* as a cashing in request. As the beneficiary is initiating the transaction, it is also the beneficiary who is paying for the transaction. Swarm per default has a check to make sure it financially makes sense to do this: currently the transaction is only started if the payout of the cheque is twice as big as the (estimated) transaction costs. 
+
+The smart contract, after doing the appropriate sanity checks (checking the validity and the funds of the issuer), will transfer the funds from the issuer's contract to the beneficiary contract. If the cheque bounced due to insufficient funds in the issuer, the peer gets disconnected. Again, it is important to understand that a peer is able to freeride up to this threshold. The issuer cannot issue a cheque if there are insufficient funds; normal operation involves a smart contract call to check for funds. If a malicious node sends a bogus, malformed or altered cheque, the cheque will bounce and thus can be identified as fraud attempt, resulting in a disconnect.
+
+Refer to the documentation for details and specification of how the protocol handles subsequent cheques and how this evolves over time.
```

Comment:
What documentation does this refer to?

Reply:
Well really the book of Swarm...which is not published yet, and in the last roundtable it was suggested that we should not refer to it yet until it's public....But I guess the swap, swear and swindle doc is enough for this for now.